libsql-wal/src/segment/current.rs

```diff
@@ -146,30 +221,29 @@ impl<F> CurrentSegment<F> {
                 tx.next_frame_no += 1;
                 tx.next_offset += 1;
             }
+        }
 
-            if let Some(size_after) = size_after {
-                if tx.not_empty() {
-                    let last_frame_no = tx.next_frame_no - 1;
-                    let mut header = { *self.header.lock() };
-                    header.last_commited_frame_no = last_frame_no.into();
-                    header.db_size = size_after.into();
-                    header.recompute_checksum();
+        if let Some(size_after) = size_after {
+            if tx.not_empty() {
+                let last_frame_no = tx.next_frame_no - 1;
+                let mut header = { *self.header.lock() };
```

Comment:
why these {}?

Reply:
Cuz I'm scared of those temporary variables now lol